communication/slack-config/sig-release/usergroups.yaml

```diff
@@ -49,17 +49,17 @@ usergroups:
       - sig-release
     members:
       - cpanato # SIG Release Technical Lead
-      - Heba # 1.27 RT Lead Shadow
-      - hosseinsalahi # 1.27 RT Lead Shadow
-      - JamesLaverack # 1.27 Emeritus Advisor
+      - gracenng # 1.28 RT Lead
+      - helayoty # 1.28 RT Lead Shadow
+      - leonardpahlke # 1.28 Emeritus Adviser
```

Comment:
small nit: this line should be added 2 lines below.

Reply:
fixed - thanks!